[Tracer][PHP] Add remark on appsec module being loaded even if not enabled (#15059)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/php.md

Restart PHP (PHP-FPM or the Apache SAPI) and visit a tracing-enabled endpoint of your application. For traces, see the [APM Service List][5].

> Note that even when not specifying `--enable-appsec`, the appsec extension will be shortly loaded at startup, but not enabled by default. It will immediately be short-circuited out, causing negligible performance overhead.

Reviewer: ```suggestion
When you do not specify `--enable-appsec`, the AppSec extension loads shortly at startup, and is not enabled by default. It immediately short-circuits, causing negligible performance overhead.
```
